lib/itamae/logger.rb

```diff
@@ -48,7 +48,7 @@ def color(code, &block)
       %w!debug info warn error fatal unknown!.each do |level|
         module_eval(<<-EOC, __FILE__, __LINE__ + 1)
           def #{level}(msg)
-            super("  " * indent_depth + msg)
+            super(format("%s%s", "  " * indent_depth, msg))
```

Comment:
If I keep original code, `itamae docker .... -l debug` is failed.  Because, Docker::Connection#log_request is call logger with Array.
#=> https://github.com/swipely/docker-api/blob/5dee4c3cadce8ff52922515749aa37e742936470/lib/docker/connection.rb#L58

Or, Isn't there a good method for that?